test002.c: 函数 `test00' :
test002.c:9: エラー: 添字参照された値が void (*)(int) です.
test002.c:10: エラー: 添字参照された値が void (*)(int) です.
test002.c: 函数 `test01' :
test002.c:16: error: member is required for not structure or union.
test002.c:16: エラー: `member' は宣言されていません.
test002.c: 函数 `test02' :
test002.c:22: error: member is required for not structure or union.
test002.c:22: エラー: `member' は宣言されていません.
test002.c: 函数 `test03' :
test002.c:28: エラー: `++' 演算子が型 `void(void) 'に適用されています.
test002.c: 函数 `test04' :
test002.c:34: エラー: `--' 演算子が型 `void(void) 'に適用されています.
test002.c: 函数 `test05' :
test002.c:40: エラー: `++' 演算子が型 `void(void) 'に適用されています.
test002.c: 函数 `test06' :
test002.c:46: エラー: `--' 演算子が型 `void(void) 'に適用されています.
test002.c: 函数 `test07' :
test002.c:52: エラー: 単項の `+' が型 `void (*)(void)' に適用されています.
test002.c: 函数 `test08' :
test002.c:58: エラー: 単項の `-' が型 `void (*)(void)' に適用されています.
test002.c: 函数 `test09' :
test002.c:64: エラー: 単項の `~' が型 `void (*)(void)' に適用されています.
test002.c: 函数 `test10' :
test002.c:70: エラー: `sizeof' が型 `void(void)' に適用されています.
test002.c: 函数 `test11' :
test002.c:76: エラー: 二項演算子 `*' が型 `void (*)(int)' と型 `const int' に適用されました.
test002.c:77: エラー: 二項演算子 `*' が型 `int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test12' :
test002.c:83: エラー: 二項演算子 `/' が型 `void (*)(int)' と型 `const int' に適用されました.
test002.c:84: エラー: 二項演算子 `/' が型 `int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test13' :
test002.c:90: エラー: 二項演算子 `%' が型 `void (*)(int)' と型 `const int' に適用されました.
test002.c:91: エラー: 二項演算子 `%' が型 `int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test14' :
test002.c:97: エラー: 二項演算子 `+' が型 `void (*)(int)' と型 `int' に適用されました.
test002.c:98: エラー: 二項演算子 `+' が型 `void (*)(int)' と型 `int' に適用されました.
test002.c: 函数 `test15' :
test002.c:104: エラー: 二項演算子 `-' が型 `void (*)(void)' と型 `const int' に適用されました.
test002.c:105: エラー: 二項演算子 `-' が型 `const int' と型 `void (*)(void)' に適用されました.
test002.c: 函数 `test16' :
test002.c:111: エラー: 二項演算子 `-' が型 `void(void)' と型 `void(void)' に適用されました.
test002.c:112: エラー: 二項演算子 `-' が型 `void (*)(void)' と型 `void (*)(void)' に適用されました.
test002.c: 函数 `test17' :
test002.c:118: エラー: 二項演算子 `<<' が型 `void (*)(int)' と型 `int' に適用されました.
test002.c:119: エラー: 二項演算子 `<<' が型 `int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test18' :
test002.c:125: エラー: 二項演算子 `>>' が型 `void (*)(void)' と型 `const int' に適用されました.
test002.c:126: エラー: 二項演算子 `>>' が型 `const int' と型 `void (*)(void)' に適用されました.
test002.c: 函数 `test19' :
test002.c:132: エラー: 二項演算子 `&' が型 `void (*)(void)' と型 `const int' に適用されました.
test002.c:133: エラー: 二項演算子 `&' が型 `const int' と型 `void (*)(void)' に適用されました.
test002.c: 函数 `test20' :
test002.c:140: エラー: 二項演算子 `^' が型 `void (*)(int)' と型 `int' に適用されました.
test002.c:141: エラー: 二項演算子 `^' が型 `int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test21' :
test002.c:147: エラー: 二項演算子 `|' が型 `void (*)(int)' と型 `int' に適用されました.
test002.c:148: エラー: 二項演算子 `|' が型 `const int' と型 `void (*)(int)' に適用されました.
test002.c: 函数 `test22' :
test002.c:156: エラー: 3 項演算子の第 2 式と第 3 式の型が不正です.
test002.c: 函数 `test23' :
test002.c:162: エラー: 代入演算子が型 `void(void)' に適用されています.
test002.c:163: エラー: 代入演算子が型 `void(void)' に適用されています.
